extension/controller/backgroundScript.js

```diff
@@ -18,14 +19,19 @@ const messageListener = async function(message, sender) {
     let webNavigationCompletedLoad = null;
     switch (message.command) {
         case "detectPageLanguage":
+            if (!modelFastText) break;
 
             /*
              * call the cld experiment to detect the language of the snippet
              * extracted from the page
              */
             languageDetection = Object.assign(new LanguageDetection(), message.languageDetection);
-            languageDetection.pageLanguage = await
-                browser.experiments.languageDetector.detect(languageDetection.wordsToDetect);
+            languageDetection.pageLanguage = modelFastText
+                .predict(languageDetection.wordsToDetect.trim().replace(/(\r\n|\n|\r)/gm, ""), 1, 0.0)
+                .get(0)[1]
+                .replace("__label__", "");
+            // ugly workaround for Norsk bokmål due errors everywhere
+            if (languageDetection.pageLanguage === "no") languageDetection.pageLanguage = "nb"
```

Comment:
please elaborate on this. will nynorsk work? is it just code replacement because it's different from our notation ("nb")? If yes, I would change this comment to state this explicitly.

Reply:
> will nynorsk work?
Yes, as @acornestean test report states above

> is it just code replacement because it's different from our notation ("nb")? If yes, I would change this comment to state this explicitly.
yes, I just changed the comment to clarify that